homeassistant/components/lacrosse_view/sensor.py

```diff
@@ -148,10 +148,10 @@ def __init__(
 
         self.entity_description = description
         self._attr_unique_id = f"{sensor.sensor_id}-{description.key}"
-        self._attr_name = f"{sensor.location.name} {description.name}"
+        self._attr_name = f"{description.name}"
```

Comment:
We don't need this line as the entity will use the description name by default if an entity description is set.
```suggestion
```